tests/test_modeling_common.py

```diff
@@ -3245,6 +3245,7 @@ def test_flash_attn_2_conversion(self):
     @require_torch_gpu
     @mark.flash_attn_test
     @slow
+    @is_flaky
```

Comment:
fa2 tests are flaky on tiny models, they should pass if one runs then 5 times, which is what `is_flaky` does